tests/testthat/helper.R

```diff
@@ -0,0 +1,19 @@
+with_mock_dir <- function(dir, ...) {
```

Comment:
Would you be interested in contributing a version of this upstream? I think this would be a significant usability improvement over the status quo (hacks listed in the FAQ)--what do you think?

Reply:
Definitely!

Reply:
https://github.com/nealrichardson/httptest/pull/48